index.html

```diff
@@ -857,6 +852,91 @@
+
+![Figure [coat_darkening_grid]: Diffuse base with a clear-coat, for coats of different coat IOR $\eta_c$](images/coat_darkening_grid.png width="60%")
+
+However this darkening may not always be desirable artistically, as in some applications it is beneficial for the observed color of the coated color to "match" the input base color (in a sense defined more precisely below in equation [undarkened_coat_albedo]). We allow for this by introducing a **`coat_darkening`** parameter, $\delta$, which defaults to 1, meaning the physically correct darkening effect due to internal reflections occurs as normal. In the case **`coat_darkening`** $\delta$ = $0$, the base albedo is instead _boosted_ (uniformly) by just enough to counteract the darkening effect. The boost factor is reduced to 1 linearly as $\delta \rightarrow 1$.
+
+What we mean by "counteract the darkening" is defined as the albedo of the coated base viewed at normal incidence, $E_c$, being equal to the "undarkened" albedo
+\begin{equation} \label{undarkened_coat_albedo}
+E_c = F_0 + E_b (1 - F_0) = \mathrm{lerp}(F_0, 1, E_b)
+\end{equation}
+where $E_b$ is the base albedo at normal incidence, which is a straightforward interpretation of the requirement for the observed color of the coated base to match the uncoated base, taking into account the presence of the Fresnel effect of the coat. Due to the physical effect of the darkening due to the internal reflections, the base BSDF would generally need to be boosted by some compensating uniform "boost" factor $B_0$ in order for the coated base to achieve this undarkened albedo. The effect of $\delta$ = **`coat_darkening`** is then defined to be the following modulation of the applied boost factor:
+\begin{equation} \label{boost_factor}
+B(\delta) = \mathrm{lerp}(B_0, 1, \delta) \ .
+\end{equation}
+While this defines the behavior at a physical level, in practice implementations will need to develop some specific approximation for the coat darkening effect.
+While this can be implementation dependent, we suggest here a reasonably simple, efficient scheme which captures the essential behavior.
+
+It can be shown that in the case of a Lambertian base with (constant) albedo $E_b$ and a perfectly smooth dielectric clear-coat -- the so-called _interfaced Lambertian_ model ([#Elias2001], [#d'Eon2021]) -- that the exact directional albedo of the coated base has the albedo-scaling-like form (where $\eta_c$ is the ratio of the coat IOR to the ambient IOR, and $F$ is the corresponding coat Fresnel factor):
+\begin{equation} \label{interfaced_lambertian_albedo}
+E_c(\omega_o) = F(\omega_o, \eta_c) + E_b \Bigl(1 - F(\omega_o, \eta_c)\Bigr) \Delta(E_b, \eta_c)
+\end{equation}
+where $\Delta(E_b, \eta_c)$ is a darkening factor given by
+\begin{equation} \label{general_darkening_formula}
+\Delta(E_b, \eta_c) = \frac{1 - K}{1 - E_b K} \ .
```

Comment:
I haven't looked at the sources yet but I'm assuming the &\Delta& terminology and definition comes from there.

Reply:
I'm also assuming this formula is the closed form of a geometric series. Is that the case?

Reply:
>I haven't looked at the sources yet but I'm assuming the $\Delta$  terminology and definition comes from there.

No actually, I just used $\Delta$ for the "darkening factor" since it's a Greek D..   Hopefully it seems reasonable anyway. For us, this darkening factor is a relevant quantity, but not really in the references.

>I'm also assuming this formula is the closed form of a geometric series. Is that the case?

Yes I think it can be derived via a geometric series (considering the "ladder" of bounces within the layer) hence the form of it, though in the Elias reference it's derived using some radiative transfer arguments so the series is not made explicit.

Here is my equations (57), (58) in the form shown in the Elias paper (which is essentially the directional albedo, excluding the Fresnel reflection):

![image](https://github.com/AcademySoftwareFoundation/OpenPBR/assets/2774364/bee3e68e-d083-425a-8848-f271577b9e34)

where for them:
  - $\rho$ is the base albedo (they assume Lambertian)
  - $r_i$ is the "internal diffuse reflection coefficient", that I call $K$.
  - $r_e$ ("external diffuse reflection coefficient") is just the average Fresnel factor.
  - $T$ is a transmission coefficient, so reduces to $1 - F$ with some Fresnel factor $F$.

Hopefully the correspondence is not too hard to make. (My notation is consistent with the more detailed write-up I did).